src/components/network-chart/NetworkChartRow.js

```diff
@@ -493,6 +493,10 @@ export class NetworkChartRow extends React.PureComponent<
         onMouseEnter={this._hoverIn}
         onMouseLeave={this._hoverOut}
         onMouseDown={this._onMouseDown}
+        aria-selected={isSelected}
+        aria-label={`${marker.name}`}
```

Comment:
I think you can simply do:
```suggestion
        aria-label={marker.name}
```
if we won't add anything more. (maybe we should, but that's another story).